Hannah-Sten/TeXiFy-IDEA
# Texlive install packages

#### Summary of additions and changes

* On TeX Live:
  * Adds inspection that checks if package is installed, and provides a quick fix to install a missing package (in the background).
* On anything not TeX Live:
  * The inspection is disabled by default and does nothing when enabled. 

#### How to test this pull request

* Use a package (with `\usepackage` or `\RequirePackage`) that is not installed and use the quick fix to install it.
* Verify that it is disabled by default on MiKTeX.
* Verify that when enabled on MiKTeX, it does nothing.

#### Wiki


- [x] Updated the wiki: https://github.com/Hannah-Sten/TeXiFy-IDEA/wiki/Probable-bugs#package-may-not-exist
